### PR TITLE
Maas 3229 add failing metrics to result meta

### PR DESCRIPTION
**Pull request by tetianakh**
Add `failing_metrics` to event meta.

Report failure per metric in prometheus plugin events.

### CODE REVIEW

File: alamo_worker/plugins/evaluate.py [hunk 1]
```diff
@@ -78,9 +88,20 @@ def _get_result_stub(status: int = RESULT_UNKNOWN,
...
+            result['failed_metrics'] = self.to_dict(evaluation_result.value)
+        return result
 
-        return {'links': result}
+    @staticmethod
+    def to_dict(value):
+        if not isinstance(value, Series):
+            return None
+        print(value)
+        print(value.values)
```
Comment (pnuckowski) [inline alamo_worker/plugins/evaluate.py]: print?

Comment (tetianakh) [reply alamo_worker/plugins/evaluate.py]: thanks, I've missed them

File: alamo_worker/plugins/tests/test_prometheus_plugin.py [hunk 2]
```diff
@@ -146,10 +142,17 @@ class TestPrometheusClient(TestCase):
...
+             ('schema', 'orders'),
+             ('table', 'images')): [1.0, 2.0],
+            (('__name__', 'ReadLatency_99percentile'),
+             ('schema', 'archive'),
+             ('table', 'images')): [3.0, 4.0],
+        }
         result = await client.get(
             'query', step=STEP, frequency=FREQUENCY)
-        self.assertEqual(list(result.data), expected)
+        self.assertEqual(result.data, expected)
```
Comment (pnuckowski) [inline alamo_worker/plugins/tests/test_prometheus_plugin.py]: assertDictEqual?

Comment (pnuckowski) [inline alamo_worker/plugins/tests/test_prometheus_plugin.py]: assertDictEqual

Comment (tetianakh) [reply alamo_worker/plugins/tests/test_prometheus_plugin.py]: It is used under the hood by the framework:
"This method will be used by default to compare dictionaries in calls to assertEqual()"
https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertDictEqual

**Pull request merged**